src/dual_model_variables.jl

```diff
@@ -176,9 +178,14 @@ function set_dual_variable_name(
     i::Int,
     ci_name::String,
     prefix::String,
+    unique_var::Bool,
```

Comment:
How about `; ensure_unique::Bool = true,`.

If this is a public method, then we don't need to break things. Appending a `false` isn't very informative for the caller.

Reply:
sounds good. It's not super important since this is internal anyway